scripts/opposed-wfrp4e.js

```diff
@@ -367,7 +387,9 @@ class OpposedWFRP
         await OpposedWFRP.evaluateOpposedTest(attacker, defender,
         {
           target: true,
-          startMessageId: actor.data.flags.oppose.startMessageId
+          startMessageId: actor.data.flags.oppose.startMessageId,
+          whisper: message.data.whisper,
+          blind: message.data.blind,
```

Comment:
Similarly, can you just pass `rollMode` and use `chatDataSetup` instead?

Reply:
In that case, it is not the rollMode activated in the GM client. It is the whisper and blind value from the modified chat message. So for me, using rollMode and chatDataSetup will not have the same result.